conwayste/src/client.rs

```diff
@@ -834,6 +828,20 @@ struct UniDrawParams {
 }
 
 impl MainState {
+    fn get_gamearea(&mut self) -> GameAreaState {
```

Comment:
Why not have this return a `Result` instead of calling `error!` here? I'd prefer to have the `error!` call closer to the top level.